test/src/server/routes/type-editor/relationship-type.js

```diff
@@ -0,0 +1,188 @@
+		expect(relationshipType.get('sourceEntityType')).to.equal(res.body.sourceEntityType);
+		expect(relationshipType.get('targetEntityType')).to.equal(res.body.targetEntityType);
+		expect(res.status).to.equal(200);
+	});
+
+	it('should be able to correctly update the relationship type and return status code 200', async () => {
+		const newRelationshipTypeData = {
+			attributeTypes: [],
+			childOrder: 0,
+			deprecated: false,
+			description: 'new descryption',
+			label: 'new label',
+			linkPhrase: 'new phrase',
+			oldAttributeTypes: [],
+			parentId: null,
+			reverseLinkPhrase: 'new reverse link phrase',
+			sourceEntityType: 'Series',
+			targetEntityType: 'Edition'
+		};
+
+		const res = await agent.post(`/relationship-type/${id1}/edit/handler`).send(newRelationshipTypeData);
+		const updatedRelationshipType = await new RelationshipType({id: id1}).fetch({require: true});
+
+		expect(updatedRelationshipType.get('id')).to.equal(res.body.id);
+		expect(updatedRelationshipType.get('description')).to.equal(res.body.description);
+		expect(updatedRelationshipType.get('label')).to.equal(res.body.label);
+		expect(updatedRelationshipType.get('linkPhrase')).to.equal(res.body.linkPhrase);
+		expect(updatedRelationshipType.get('reverseLinkPhrase')).to.equal(res.body.reverseLinkPhrase);
+		expect(updatedRelationshipType.get('sourceEntityType')).to.equal(res.body.sourceEntityType);
+		expect(updatedRelationshipType.get('targetEntityType')).to.equal(res.body.targetEntityType);
+		expect(res.status).to.equal(200);
+	});
+});
+
+
+describe('Relationship Type routes without Relationship Editor priv', () => {
+	const id1 = 1;
+	let agent;
+	before(async () => {
+		try {
+			await createRelationshipType(id1);
+			await createEditor(123456, 1);
+		}
+		catch (error) {
+			// console.log(error);
+		}
+		// Log in; use agent to use logged in session
+		agent = await chai.request.agent(app);
+		await agent.get('/cb');
+	});
+	after(truncateEntities);
+
+	it('should throw an error if creating new relationship type', async () => {
+		const res = await agent.get('/relationship-type/create');
+		expect(res.ok).to.be.false;
+		expect(res).to.have.status(403);
+		expect(res.res.statusMessage).to.equal('You do not have the privilege to access this route');
+	});
+
+	it('should throw an error trying to edit an existing relationship type', async () => {
+		const relationshipTypeId = 1;
+		const res = await agent.get(`/relationship-type/${relationshipTypeId}/edit`);
+		expect(res.ok).to.be.false;
+		expect(res).to.have.status(403);
+		expect(res.res.statusMessage).to.equal('You do not have the privilege to access this route');
+	});
+
+	it('should throw a 404 error when trying to edit a non-existent relationship type', async () => {
+		const relationshipTypeId = 222;
+		const res = await agent.get(`relationship-type/${relationshipTypeId}/edit`);
+		expect(res.ok).to.be.false;
+		expect(res).to.have.status(404);
```

Comment:
Same missing`/` issue as above.
Also, without the correct privs, we should get a 403 response *before* we get to the point where the rel type is loaded from the DB (which would 404 in this case)
```suggestion
	it('should throw a 403 error when trying to edit a non-existent relationship type', async () => {
		const relationshipTypeId = 222;
		const res = await agent.get(`/relationship-type/${relationshipTypeId}/edit`);
		expect(res.ok).to.be.false;
		expect(res).to.have.status(403);
```